binsync/common/ui/force_push/force_push.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+import binsync.data
+from binsync.common.ui.force_push.panels.functions_table import QFunctionTable
+from binsync.common.ui.force_push.panels.global_panel import QGlobalsTable
+from binsync.common.ui.qt_objects import (
+    QLabel,
+    QStatusBar,
+    QTabWidget,
+    QVBoxLayout,
+    QWidget,
+    Signal
+)
+
+l = logging.getLogger(__name__)
+
+class ForcePushUI(QWidget):
+    update_ready = Signal()
+    def __init__(self, controller, parent=None):
+        super(ForcePushUI, self).__init__(parent)
+        self.controller = controller
+
+        self.tables = {}
+        self._init_widgets()
+
+        # register controller callback
+        self.update_ready.connect(self.reload)
```

Comment:
Making a callback for this UI is redundant and causes the UI to freeze every 10 seconds. We don't need to update this UI every binsync_ui_callback, because the functions available to Force Push **NEVER** change... think about that.